src/mono/wasi/build/WasiApp.Native.targets

```diff
@@ -341,9 +341,9 @@
           <!--EnvironmentVariables="@(EmscriptenEnvVars)" />-->
   <!--</Target>-->
 
-  <UsingTask TaskName="Microsoft.WebAssembly.Build.Tasks.EmitWasmBundleObjectFiles" AssemblyFile="$(WasmAppBuilderTasksAssemblyPath)" />
+  <UsingTask TaskName="EmitBundleObjectFiles" AssemblyFile="$(EmitBundleTasksAssemblyPath)" />
```

Comment:
This runs on user machine. The property needs to be set also in the [.\src\mono\nuget\Microsoft.NET.Runtime.WebAssembly.Wasi.Sdk\Sdk\Sdk.targets.in](https://github.com/dotnet/runtime/blob/main/src/mono/nuget/Microsoft.NET.Runtime.WebAssembly.Wasi.Sdk/Sdk/Sdk.targets.in) and for consistency probably even in the [.\src\mono\nuget\Microsoft.NET.Runtime.WebAssembly.Sdk\Sdk\Sdk.targets.in](https://github.com/dotnet/runtime/blob/main/src/mono/nuget/Microsoft.NET.Runtime.WebAssembly.Sdk/Sdk/Sdk.targets.in)